homeassistant/components/vicare/climate.py

```diff
@@ -134,6 +144,40 @@ def update(self):
             "circulationpump_active"
         ] = self._api.getCirculationPumpActive()
 
+        # Update the specific device attributes
+        if self._heating_type == HeatingType.gas:
+            self._attributes["burner_active"] = self._api.getBurnerActive()
```

Comment:
Is this a binary value? Then it should probably be a binary sensor.

Reply:
Shouldn't this be implemented as hvac_action? It basically indicates if the burner is idle or heating.

Reply:
Yes, that's also a possibility. Then we can also remove it from state attributes here.

Reply:
@crazyfx1 is there a similar thing for heat pumps as well?

Reply:
Yes, it's the getCompressorActive() for heat pumps.

Reply:
Now mapped to hvac action.
@oischinger I hope thats the way you intended. We might add some logic later to distinguish between "Idle" and "Off" by checking the current mode.